appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+
+ //needed to communicate/translate between JSONWire cookies and regular JS cookies 
```

Comment:
when we have a line comment like this, we always have one space between the comment symbol and the comment, as follows:

```js
// needed to communicate ...
```